packages/@react-spectrum/textfield/stories/Textfield.stories.js

```diff
@@ -140,6 +140,21 @@ storiesOf('TextField', module)
   )
   .add('custom width small, labelPosition: side',
     () => render({icon: <Info />, validationState: 'invalid', width: '30px', labelPosition: 'side'})
+  )
```

Comment:
Mind adding stories for validation states (invalid, valid), help text (description/error message), icons, isRequired, isDisabled

For followup later, will be simpler to add `controls` to toggle these various things on and off

Reply:
do you mean just in general creating more stories that use a combination of different states or specifically for read only since i believe read only won't have icons, validation, or help text

Reply:
right, and we'll want to verify that those show up as we expect them to

Reply:
Ah right, I forgot that we'd be dropping icon/validation icons for readonly. It would be good to have more read only stories with the above states just so we can verify that the correct behavior happens (e.g. no icons appear). May be a good idea to make these chromatic stories since they are visual for the most part

Reply:
ohhh okay i see what you mean. yeah i can definitely add more stories